playground/app.jsx

```diff
@@ -211,7 +211,7 @@ const App = ({location: {pathname, query}}) => (
               id="test"
               value="purrr"
               query={query}
-              onGetSuggestions={() => {}}
+              onSelectSuggestion={suggestion => { console.log('You have chosen ', suggestion); }}
```

Comment:
This is causing the tests to fail because console.logs are not allowed (committed console logs :) )